pkgs/by-name/xc/xcbuild/patches/Use-system-toolchain-for-usr-bin.patch

```diff
@@ -12,12 +11,17 @@ index 9d6d4576d7..73aabc3d42 100644
  using libutil::Filesystem;
  using libutil::FSUtil;
  
-+#define SYSTEM_DEVELOPER_DIR "/private/var/select/developer_dir"
++namespace {
++  const std::vector<const std::string> kSystemDeveloperDirs = {
++    "/private/var/select/developer_dir",
++    "/private/var/db/xcode_select_link"
++  };
++}
```

Comment:
I think it's possible to make this a function and add the availability attribute to it, but I personally prefer checking every possible path because it doesn't depend on knowing which macOS release does what.